File: 31 ObjC/31 ObjC/Person.m
//
//  Person.m
//  31 ObjC
//
//  Created by Johan Wigmo on 2019-01-31.
//  Copyright © 2019 Newton. All rights reserved.
//

#import "Person.h"

@implementation Person

- (instancetype)initWithName: (NSString *)name age: (int)age andNickname: (nullable NSString*)nickname
{
    self = [super init];
    if (self) {
        self.name = name;
        self.age = age;
        self.nickname = nickname;
    }
    return self;
}

@end

File: 07_Repetition/07 Repetition/Person.swift
//
//  Person.swift
//  07 Repetition
//
//  Created by Johan Wigmo on 2019-01-07.
//  Copyright © 2019 Johan Wigmo. All rights reserved.
//

import UIKit

struct Person {
    
    let firstName: String
    let lastName: String
    let age: Int
    let image: UIImage?
    
}

extension Person {
    
    var fullName: String {
        return firstName + " " + lastName
    }
    
}
